pkg/volume/glusterfs/glusterfs.go

```diff
@@ -68,6 +76,10 @@ const (
 	annGlusterUserKey           = "glusterfs.kubernetes.io/userkey"
 	annGlusterUser              = "glusterfs.kubernetes.io/userid"
 	gciGlusterMountBinariesPath = "/sbin/mount.glusterfs"
+	defaultGidMin               = "2000"
+	defaultGidMax               = "4294967295"
+	annDynamicallyProvisioned   = "pv.kubernetes.io/provisioned-by"
```

Comment:
shouldn't need this annotation... we should pick up all gids on PVs related to a storage class we're responsible for allocating from, regardless of who provisioned it